### PR TITLE
Implemented new docs structure

### PR DESCRIPTION
**Pull request by tusharojha**
## Implemented new docs structure

It includes a new header with tabs like this:
Screenshot 2022-07-12 at 9 15 23 AM

The sidebar now auto-adjusts according to the selected section from header. 

We are using 4 different types of sidebars in `sidebars.js` file. These are:

- homeSidebar
- tutorialsSidebar
- developSidebar
- announcementSidebar

### CODE REVIEW

File: docs/announcements/Domains.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+id: domains
+title: Dotsama Domains
+displayed_sidebar: announcementSidebar
+---
+
+We are happy to launch dotsama domains.
```
Comment (Yung-Beef) [inline docs/announcements/Domains.md:7]:
```suggestion
We are excited to launch Dotsama Domains!
```

File: docs/basics/lightpaper/beyond-v1.0/AdvancedModeration.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 id: advanced-moderation
-title: Advanced Moderation
+title: Advanced moderation
```
Comment (Yung-Beef) [inline docs/basics/lightpaper/beyond-v1.0/AdvancedModeration.md:3]:
```suggestion
title: Advanced Moderation
```

File: docs/basics/lightpaper/beyond-v1.0/LookingBeyondV1.0.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 id: looking-beyond-v1.0
-title: Looking Beyond v1.0
+title: Looking beyond v1.0
```
Comment (Yung-Beef) [inline docs/basics/lightpaper/beyond-v1.0/LookingBeyondV1.0.md:3]:
```suggestion
title: Looking Beyond v1.0
```

File: docs/basics/lightpaper/beyond-v1.0/Marketplace.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 id: marketplace
-title: Algorithm Marketplace
+title: Marketplace for social networking algorithms
```
Comment (Yung-Beef) [inline docs/basics/lightpaper/beyond-v1.0/Marketplace.md:3]:
```suggestion
title: Marketplace For Content Algorithms
```

File: docs/basics/lightpaper/beyond-v1.0/SplitReputation.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 id: split-reputation
-title: Split Reputation
+title: Split reputation
```
Comment (Yung-Beef) [inline docs/basics/lightpaper/beyond-v1.0/SplitReputation.md:3]:
```suggestion
title: Split Reputation
```

File: docs/develop/Introduction.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+### Welcome Devs 👋🏼
+
+This is your cool place to hangout and build on Subsocial 🧑🏻‍💻
+
+The documentation is divided into several subsections for quick access and fast building, here's the structure:
+
+| Resource      | Description |
+| ----------- | ----------- |
+| [Cheatsheet / Quick Reference](/docs/develop/quick-reference)      | Quick easy way to get started with the most common uses of Subsocial JS SDK.       |
```
Comment (Yung-Beef) [inline docs/develop/Introduction.md:16]:
```suggestion
| [Cheatsheet / Quick Reference](/docs/develop/quick-reference)      | A quick and easy way to get started with the most common uses of the Subsocial JS SDK.       |
```

File: docs/develop/Introduction.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+| [How-to-Guides](/docs/develop/how-to-guides/connectToSubsocial)   | Contains detailed information about how to do different operations on Subsocial blockchain.        |
```
Comment (Yung-Beef) [inline docs/develop/Introduction.md:17]:
```suggestion
| [How-to-Guides](/docs/develop/how-to-guides/connectToSubsocial)   | Contains detailed information about how to do different operations on the Subsocial blockchain.        |
```

File: docs/develop/Introduction.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+| [TypeDocs](https://docs.subsocial.network/js-docs/js-sdk/index.html)   | Documentation created from the codebase itself.        |
+| [Playground](https://github.com/tusharojha/playground)   | Run Subsocial SDK code on the air without downloading anything.        |
+| [Examples](https://github.com/dappforce?q=example&type=all&language=&sort=)   | Beginner friendly templates & examples for building on Subsocial.       |
+| [YouTube Tutorials](https://youtu.be/jeTrpltqAqc)   | Watch some video and get started building along with it.      |
```
Comment (Yung-Beef) [inline docs/develop/Introduction.md:21]:
```suggestion
| [YouTube Tutorials](https://youtu.be/jeTrpltqAqc)   | Watch some videos and start building along with them.      |
```

File: docs/develop/getting-started/TypesOfAPIs.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
-id: thebasics
-title: The Basics
+id: typesOfApis
+title: Types of APIs
```
Comment (Yung-Beef) [inline docs/develop/getting-started/TypesOfAPIs.md:3]: Would "API Types" be better?

File: docs/develop/getting-started/UnderstandingTestnet.md [hunk 10]
```diff
@@ -1,8 +1,11 @@
 ---
 id: testnet
-title: About SubSocial Testnet
+title: Understanding Testnet
```
Comment (Yung-Beef) [inline docs/develop/getting-started/UnderstandingTestnet.md:3]:
```suggestion
title: Understanding The Testnet
```

File: docs/develop/getting-started/UnderstandingTestnet.md [hunk 11]
```diff
@@ -1,8 +1,11 @@
...
+displayed_sidebar: developSidebar
 ---
 
+**This section covers information about Subsocial Testnet i.e. SoonSocialX.**
```
Comment (Yung-Beef) [inline docs/develop/getting-started/UnderstandingTestnet.md:7]:
```suggestion
**This section covers information about Subsocial's testnet: SoonSocialX.**
```

File: docs/develop/how-to-guides/ConnectToSubsocial.md [hunk 12]
```diff
@@ -1,7 +1,10 @@
 ---
-id: gettingstarted
-title: Getting Started
+id: connectToSubsocial
+title: Connect to Subsocial
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/ConnectToSubsocial.md:3]:
```suggestion
title: Connect To Subsocial
```

File: docs/develop/how-to-guides/ConnectToSubsocial.md [hunk 13]
```diff
@@ -1,7 +1,10 @@
...
+displayed_sidebar: developSidebar
 ---
+**This section covers how to connect to Subsocial blockchain using our JS SDK.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/ConnectToSubsocial.md:6]:
```suggestion
**This section covers how to connect to the Subsocial blockchain using our JS SDK.**
```

File: docs/develop/how-to-guides/Structures.md [hunk 14]
```diff
@@ -1,30 +1,39 @@
...
+id: structures
+title: Blockchain Structures
+displayed_sidebar: developSidebar
 ---
-Subsocial uses a few building blocks to enable social networking. A short summary of each is provided below.
+**This section covers how spaces, posts and reactions are related to each other.**
+
+Subsocial uses a few building blocks to enable social networking. 
+
+A quick glimpse of how the major components i.e. Space, Post and Reactions are related.
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/Structures.md:10]:
```suggestion
Here we will take a quick glimpse of how the major components such as Spaces, Posts, and Reactions are related.
```

File: docs/develop/how-to-guides/Structures.md [hunk 15]
```diff
@@ -1,30 +1,39 @@
 ---
-id: parts
-title: What Does Subsocial Consist Of?
+id: structures
+title: Blockchain Structures
+displayed_sidebar: developSidebar
 ---
-Subsocial uses a few building blocks to enable social networking. A short summary of each is provided below.
+**This section covers how spaces, posts and reactions are related to each other.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/Structures.md:6]:
```suggestion
**This section covers how spaces, posts, and reactions are related to each other.**
```

File: docs/develop/how-to-guides/Transactions.md [hunk 16]
```diff
@@ -1,8 +1,11 @@
 ---
 id: transactions
-title: Transactions
+title: Signing & Sending Transactions
+displayed_sidebar: developSidebar
 ---
 
+**This section covers how to sign and send transactions to Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/Transactions.md:7]:
```suggestion
**This section covers how to sign and send transactions on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/comments/CreatingComments.md [hunk 17]
```diff
@@ -1,7 +1,9 @@
 ---
-id: creating-comments
-title: Creating And Updating Comments
+id: create-comments
+title: Create and update Comments
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/comments/CreatingComments.md:3]:
```suggestion
title: Create And Update Comments
```

File: docs/develop/how-to-guides/comments/CreatingComments.md [hunk 18]
```diff
@@ -1,7 +1,9 @@
...
+displayed_sidebar: developSidebar
 ---
+**This section covers how to create and update comments on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/comments/CreatingComments.md:6]:
```suggestion
**This section covers how to create and update comments on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/comments/GettingComments.md [hunk 19]
```diff
@@ -1,7 +1,9 @@
 ---
-id: getting-comments
-title: Getting Comments
+id: fetch-comments
+title: Fetch Comments
+displayed_sidebar: developSidebar
 ---
+**This section covers how to create and update comments on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/comments/GettingComments.md:6]:
```suggestion
**This section covers how to fetch comments on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/follow/CreatingFollow.md [hunk 20]
```diff
@@ -1,7 +1,10 @@
 ---
-id: following
+id: create-follow
 title: Following And Unfollowing
+displayed_sidebar: developSidebar
 ---
+**This section covers how to follow and unfollow on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/follow/CreatingFollow.md:6]:
```suggestion
**This section covers how to follow and unfollow on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/follow/GettingFollow.md [hunk 21]
```diff
@@ -1,7 +1,10 @@
 ---
-id: getting-follow
-title: Getting Follows
+id: fetch-follow
+title: Fetch Follows
+displayed_sidebar: developSidebar
 ---
+**This section covers how to fetch follows on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/follow/GettingFollow.md:6]:
```suggestion
**This section covers how to fetch follows on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/posts/CreatingPosts.md [hunk 22]
```diff
@@ -1,7 +1,9 @@
 ---
-id: creating-posts
-title: Creating And Updating Posts
+id: create-posts
+title: Create and update Posts
+displayed_sidebar: developSidebar
 ---
+**This section covers how to create and update posts on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/posts/CreatingPosts.md:6]:
```suggestion
**This section covers how to create and update posts on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/posts/FetchingPosts.md [hunk 23]
```diff
@@ -1,7 +1,9 @@
 ---
-id: getting-posts
-title: Getting Posts
+id: fetch-posts
+title: Fetch Posts
+displayed_sidebar: developSidebar
 ---
+**This section covers how to fetch posts from Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/posts/FetchingPosts.md:6]:
```suggestion
**This section covers how to fetch posts from the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/profiles/CreatingProfiles.md [hunk 24]
```diff
@@ -1,8 +1,11 @@
 ---
-id: creating-profiles
-title: Creating And Updating Profiles
+id: create-profiles
+title: Create and update Profiles
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/profiles/CreatingProfiles.md:3]:
```suggestion
title: Create And Update Profiles
```

File: docs/develop/how-to-guides/profiles/CreatingProfiles.md [hunk 25]
```diff
@@ -1,8 +1,11 @@
...
+displayed_sidebar: developSidebar
 ---
 
+**This section covers how to create and update profiles on Subsocial Blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/profiles/CreatingProfiles.md:7]:
```suggestion
**This section covers how to create and update profiles on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/profiles/FetchingProfiles.md [hunk 26]
```diff
@@ -1,7 +1,11 @@
 ---
-id: getting-profiles
-title: Getting Profiles
+id: fetch-profiles
+title: Fetch Profiles
+displayed_sidebar: developSidebar
 ---
+
+**This section covers how to fetch profiles from Subsocial Blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/profiles/FetchingProfiles.md:7]:
```suggestion
**This section covers how to fetch profiles from the Subsocial Blockchain.**
```

File: docs/develop/how-to-guides/reactions/CreatingReactions.md [hunk 27]
```diff
@@ -1,7 +1,9 @@
 ---
-id: creating-reactions
-title: Creating, Updating, And Deleting Reactions
+id: create-reactions
+title: Create, update, and delete Reactions
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/reactions/CreatingReactions.md:3]:
```suggestion
title: Create, Update, And Delete Reactions
```

File: docs/develop/how-to-guides/reactions/CreatingReactions.md [hunk 28]
```diff
@@ -1,7 +1,9 @@
...
+displayed_sidebar: developSidebar
 ---
+**This sections covers how to create, update and delete reactions on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/reactions/CreatingReactions.md:6]:
```suggestion
**This sections covers how to create, update and delete reactions on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/reactions/GettingReactions.md [hunk 29]
```diff
@@ -1,7 +1,9 @@
 ---
-id: getting-reactions
-title: Getting Reactions
+id: fetch-reactions
+title: Fetch Reactions
+displayed_sidebar: developSidebar
 ---
+**This sections covers how to fetch reactions on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/reactions/GettingReactions.md:6]:
```suggestion
**This sections covers how to fetch reactions on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/spaces/CreatingSpaces.md [hunk 30]
```diff
@@ -1,7 +1,10 @@
 ---
-id: creating-spaces
-title: Creating And Updating Spaces
+id: create-spaces
+title: Create and update Spaces
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/spaces/CreatingSpaces.md:3]:
```suggestion
title: Create And Update Spaces
```

File: docs/develop/how-to-guides/spaces/CreatingSpaces.md [hunk 31]
```diff
@@ -1,7 +1,10 @@
...
+displayed_sidebar: developSidebar
 ---
+**This sections covers how to create and update space on Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/spaces/CreatingSpaces.md:6]:
```suggestion
**This sections covers how to create and update spaces on the Subsocial blockchain.**
```

File: docs/develop/how-to-guides/spaces/FetchingSpaces.md [hunk 32]
```diff
@@ -1,7 +1,9 @@
 ---
-id: getting-spaces
-title: Getting Spaces
+id: fetch-spaces
+title: Fetch Spaces
+displayed_sidebar: developSidebar
 ---
+**This sections covers how to fetch space from Subsocial blockchain.**
```
Comment (Yung-Beef) [inline docs/develop/how-to-guides/spaces/FetchingSpaces.md:6]:
```suggestion
**This sections covers how to fetch spaces from the Subsocial blockchain.**
```

File: docusaurus.config.js [hunk 33]
```diff
@@ -31,6 +31,14 @@ module.exports = {
     prism: {
       theme: require('prism-react-renderer/themes/dracula'),
     },
+    announcementBar: {
+      id: 'support_us',
+      content:
+        'We are looking to revamp our docs, please fill <a target="_blank" rel="noopener noreferrer" href="https://forms.gle/2FHGBma3JPvukyeo6">this survey</a>',
```
Comment (Yung-Beef) [inline docusaurus.config.js:37]:
```suggestion
        'We are looking to revamp our docs, please fill out <a target="_blank" rel="noopener noreferrer" href="https://forms.gle/2FHGBma3JPvukyeo6">this survey</a> with any suggestions.',
```

**Review by Yung-Beef — APPROVED** [review]
Make sure the file names got updated when you updated the id and Title.

File: docs/develop/getting-started/TypesOfAPIs.md (hunk 9 above)
Comment (tusharojha) [reply docs/develop/getting-started/TypesOfAPIs.md:3]: Yup

**Review by olehmell — APPROVED** [review]

**Pull request merged**